### ISSUE TITLE
The valid path "/*" does not resolve correctly

### ISSUE THREAD

**Issue by thesunny** [dividab/tsconfig-paths#113]
Thank you to the contributors for providing this useful library. I know a lot of works goes into this. I've been trying to migrate to typescript and this issue is causing a hangup. My code follows this path setup and with so many files, I don't want to have to rewrite all my import statements.

The following is a valid path in `tsconfig.json` but will not work with tsconfig-paths. The broken part is `/*`:

```ts
{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "/*": ["*"]
    }
  }
}
```

The config above does work properly in VSCode and imports are followed correctly. It maps "/lib/something" for example to "{root}/lib/something"

In my environment, the following `tsconfig.json` with a tilde works fine with tsconfig-paths so it's likely not a configuration issue:

```ts
{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "~*": ["*"]
    }
  }
}
```

With the `tsconfig.json` above, tsconfig-paths works correctly.

When I use the "/*" path, I get an error like this:

```text
Error: Cannot find module '/lib/log'
```

**Comment by thesunny** [comment]
For clarity, this typescript PR shows that it works in typescript now: https://github.com/microsoft/TypeScript/pull/27980

**Comment by 0x80** [comment]
I just ran into the same issue. Good to know that ~/ works for now as a workaround for now 👍

**Comment by cspotcode** [comment]
Looks like the same as #128

**Comment by thesunny** [comment]
Yes, looks the same. But for clarity, this issue was opened first and the other is a duplicate. :)

**Comment by cspotcode** [comment]
@thesunny yes, definitely.  The other issue links to the bit of code that needs to be changed.  Honestly I wanted to link them together so I remember to close both when the bug is fixed.

**Comment by jonaskello** [comment]
Fixed in #180
